### ISSUE TITLE
Timeline trouble: in P12S P2, but alway show P1 timeline

### ISSUE THREAD

**Issue by Minae07** [quisquous/cactbot#5897]
My FFXIV's launcher language is JP
ACT is JP too
Version 2.9.6.4 was still working fine. After updating to 2.9.6.5 and up to the present, the timeline has consistently been at p1
So currently, P2 is also unable to report the mechanism, but P1 is functioning normally
I don't know why

**Comment by valarnin** [comment]
There are two possible ways for the timeline to sync to P2.

The first is a sync on an AddCombatant line, e.g.:
```
03|2023-11-02T22:44:55.8160000-04:00|400072A7|Hemitheos|00|5A|0000|00||12383|16182|69200|69200|10000|10000|||99.89|90.48|0.00|0.00|beb4d039d5e29797
```
See:
https://github.com/quisquous/cactbot/blob/c3214ca7bd959bbd35074c9c6a5a3c88fc05002c/ui/raidboss/data/06-ew/raid/p12s.txt#L14

The second is via the first `Ultima` cast, e.g.:
```
20|2023-11-02T22:45:02.2530000-04:00|40006F33|Pallas Athena|8682|Ultima|40006F33|Pallas Athena|4.700|100.00|80.00|0.00|0.00|7b336c9bfa800e35
```
See:
https://github.com/quisquous/cactbot/blob/c3214ca7bd959bbd35074c9c6a5a3c88fc05002c/ui/raidboss/data/06-ew/raid/p12s.txt#L117

Can you ensure that one or both of those lines are present in your network log file? `Pallas Athena` should also match an actor name of `パラスアテナ`, and `Hemitheos` should also match an actor name of `ヘーミテオス`.

**Comment by Minae07** [comment]
> There are two possible ways for the timeline to sync to P2.
> 
> The first is a sync on an AddCombatant line, e.g.:
> 
> ```
> 03|2023-11-02T22:44:55.8160000-04:00|400072A7|Hemitheos|00|5A|0000|00||12383|16182|69200|69200|10000|10000|||99.89|90.48|0.00|0.00|beb4d039d5e29797
> ```
> 
> See:
> 
> https://github.com/quisquous/cactbot/blob/c3214ca7bd959bbd35074c9c6a5a3c88fc05002c/ui/raidboss/data/06-ew/raid/p12s.txt#L14
> 
> The second is via the first `Ultima` cast, e.g.:
> 
> ```
> 20|2023-11-02T22:45:02.2530000-04:00|40006F33|Pallas Athena|8682|Ultima|40006F33|Pallas Athena|4.700|100.00|80.00|0.00|0.00|7b336c9bfa800e35
> ```
> 
> See:
> 
> https://github.com/quisquous/cactbot/blob/c3214ca7bd959bbd35074c9c6a5a3c88fc05002c/ui/raidboss/data/06-ew/raid/p12s.txt#L117
> 
> Can you ensure that one or both of those lines are present in your network log file? `Pallas Athena` should also match an actor name of `パラスアテナ`, and `Hemitheos` should also match an actor name of `ヘーミテオス`.

I apologize for the inconvenience, I have a hard time following the text due to technical difficulties, is there a simple way to resolve this or a guide that I can follow?

**Comment by quisquous** [comment]
Can you upload a Network log of this fight? https://github.com/quisquous/cactbot/blob/main/docs/FAQ-Troubleshooting.md#how-to-find-a-network-log

**Comment by Minae07** [comment]
> Can you upload a Network log of this fight? https://github.com/quisquous/cactbot/blob/main/docs/FAQ-Troubleshooting.md#how-to-find-a-network-log

[Network_26906_20231110.zip](https://github.com/quisquous/cactbot/files/13311976/Network_26906_20231110.zip)

here ! thanks !

**Comment by valarnin** [comment]
Hmm, the actor name is missing entirely, and there's no `03` add line for the `Hemitheos` actors at all (`Add`/`Change`/`Remove` events for `261` lines are still present, so `FFXIV_ACT_Plugin` must be filtering those out for some reason? `Hemitheos`/`ヘーミテオス` `BNpcNameID` = `305F`)

```
261|2023-11-10T02:56:32.0180000+08:00|Add|40012DC9|BNpcID|3F36|BNpcNameID|305F|CastTargetID|E0000000|CurrentMP|10000|CurrentWorldID|65535|Heading|0.0000|Level|90|MaxHP|69200|MaxMP|10000|ModelStatus|18432|NPCTargetID|E0000000|PosX|99.8923|PosY|90.4813|Radius|1.7000|Type|2|WorldID|65535|18f5af070ded9d90
Regex search for `^0[34]\|.*40012DC9` shows no hits
```

Same thing for actor ID `40012D52`, which is the main `Pallas Athena`/`パラスアテナ` actor for that pull, e.g. the one all the players are hitting. No `03` or `04` lines for the actor, and the `Name` field is never set in game memory according to `261` lines, so any `20`/`21`/`22` lines involving the actor have a blank name field.

I'm not sure on this one, really looks like a weird interaction in the actual game client or something. Dalamud is present, as indicated by `Sonar` and `Orchestrion` plugin chat lines, so maybe it's an issue with a Dalamud/XIVLauncher plugin? I don't see the normal list of XL plugins dumped to echo chat though, so I'm not sure what other plugins may be present.

(I'd also expect a log like this to be rejected from FFLogs due to missing `03`/`04` lines and missing non-player combatant names, though I'm not 100% sure how sensitive their algorithms are to that)

**Comment by quisquous** [comment]
Yeah, agreed, this seems like either some Dalamud plugin is interfering, an FFXIV Plugin bug, or some weird game interaction.

Does this always happen, or was this a one-time thing? Do you know other people this is happening to? If this is happening every time, it might be worth trying without Dalamud as an experiment to at least rule one possibility out.

**Comment by valarnin** [comment]
(for further context here, the server itself sends JP lang names to all clients regardless of the client's region/language settings as part of the `SpawnNPC` packet, and that should theoretically get loaded directly into the actor struct in memory before any `BNpcNameID` language translation override happens, so I'm really not sure how/why this could happen)

**Comment by Minae07** [comment]
well... if the issue is related to Dalamud, it's strange when only P12S is affected. I did install Sonar and Orchestrion, but the timeline was functioning normally before.

**Comment by quisquous** [comment]
I guess the other thing you could do is next time you are there use the `(DEBUG) Log all Network Packets` option in the ffxiv plugin, and upload another log and we could investigate further.
